pkg/datapath/linux/node.go

```diff
@@ -716,14 +716,14 @@ func (n *linuxNodeHandler) insertNeighbor(ctx context.Context, newNode *nodeType
 	if nextHopIsNew || refresh {
 		linkAttr, err := netlink.LinkByName(ifaceName)
 		if err != nil {
-			scopedLog.WithError(err).Error("Failed to retrieve iface by name (netlink)")
+			scopedLog.WithError(err).Info("Failed to retrieve iface by name (netlink)")
```

Comment:
Same here? This seems to mean that a interface is no longer in the system? Sounds like an error to me. Have we seen these in the logs?

Reply:
This got restructured. In the case of missing link, the error will be returned to the daemon which will make the agent to panic.